src/04-modality-specific-files/04-intracranial-electroencephalography.md

```diff
@@ -352,6 +384,15 @@ SHOULD be present:
 | group           | RECOMMENDED           | The group that the electrode is a part of. Note that any group specified here should match a group specified in `_channels.tsv`. |
 | hemisphere      | RECOMMENDED           | The hemisphere in which the electrode is placed, one of `['L' or 'R']` (use capital).                                            |
 
+{{ MACROS___make_columns_table(
+   {
+      "material": "RECOMMENDED",
+      "manufacturer": "RECOMMENDED",
+      "group": "RECOMMENDED",
```

Comment:
This is the electrodes.tsv table, but it says:

> Note that any groups specified in _electrodes.tsv must match those present here.

The reference should be to channels.tsv instead